apluslms_roman/cli.py

```diff
@@ -14,8 +14,9 @@
 
 from . import __version__
 from .builder import Engine
-from .configuration import CourseConfigError, CourseConfig
-from .settings import RomanSettings
+from .configuration import (ProjectConfigError, ProjectConfig, LEGACY_CONFIG, DEFAULT_CONFIG,
```

Comment:
Write all imported keys on their own lines and sort them (though, globals in start(. So:
```
from .configuration import (
  DEFAULT_CONFIG,
  LEGACY_CONFIG,
  ProjectConfig,
  ProjectConfigError,
  is_legacy_configuration, # don't forget this comma
)
´``